docs/sdk-docs/installing-dapp/README.md

```diff
@@ -0,0 +1,40 @@
+# Install and Run the dApp
+
+Clone the Raiden Light Client repository.
+
+```bash
+git clone --recurse-submodules https://github.com/raiden-network/light-client.git
+```
+
+## Build the SDK
+
+Change to the `raiden-ts` directory, install the dependencies and build the SDK.
+
+```bash
+cd ../raiden-ts
+npm install
+npm run build
+```
+
+## Install the dApp dependencies
+
+Change to the `raiden-dapp` directory and install its dependencies.
+
+```bash
+cd ../raiden-dapp
+npm install --save raiden-ts
```

Comment:
Isn't `npm install` enough? `--save` adds it to `package.json` where it already is

Reply:
Yes, your right I've always used only npm install and it works fine.

Reply:
This was required at least once in the past due to npm link shenanigans. I am not sure if it is need anymore or if it is required only the first time you clone the project. Or the first time you start in a new environment. In my experience so far, if you set up the link once then it pretty much works normally. But if you try a new system that never had the project issues might appear.

A good way to check would be in a VM that never worked with the light client to ensure that there is no leftover setup. 

These instructions target someone that never worked with the SDK before so we have to make sure that they work in such an environment.

Reply:
I will leave this one then. I usually clone the repo and install the all dependencies  from scratch each time I start something new on the LC, dunno if that is the same as installing it on another machine or VM.

Reply:
I never ran it that way, but yes we should create a ticket and go through the steps on a clean machine and verify that they work (or not)